streamparse/cli/submit.py

```diff
@@ -205,6 +205,17 @@ def submit_topology(name=None, env_name="prod", workers=2, ackers=2,
             if isinstance(inner_shell, ShellComponent):
                 if 'streamparse_run' in inner_shell.execution_command:
                     inner_shell.execution_command = streamparse_run_path
+                    
+    # Additional options
+    additional_options = env_config.get('options', None)
+    if additional_options and isinstance(additional_options, dict):
```

Comment:
Please remove these `isinstance` checks, as I'd rather see an Exception get raised when someone has something incorrect in their `config.json` than silently ignoring it.